cli/yara.c

```diff
@@ -1044,7 +1044,13 @@ static int callback(
       mutex_unlock(&output_mutex);
     }
 
-    return CALLBACK_CONTINUE;
+      return CALLBACK_CONTINUE;
+
+    case CALLBACK_MSG_SCAN_WARNING_MATCH_LIMIT:
+      printf("Warning: maximum matches for string %s. Results may be invalid.\n",
```

Comment:
Does it make sense to send this to stderr instead of stdout?

Reply:
Yep, I think that stderr would be better.